INSTALL.rst

```diff
@@ -0,0 +1,186 @@
+###########
+Cycamore Installation Guide
+###########
+
+Cycamore uses the CMake cross platform bailed-generator as a build system. To
+simplify the usage of cmake to build Cycamore from source, a python script is
+provided with the source.
+
+.. contents:: Table of Contents
+
+******************************
+Dependencies
+******************************
+
+A full list of the Cycamore package dependencies is shown below:
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`CMake`                2.8
+`boost`                1.46.1
+`libxml2`              2
+`Cyclus`               1.4
+====================   ==================
+
+
+
+******************************
+Installation
```

Comment:
DEPENDENCIES.rst includes alternative installation instructions. Perhaps those should go here?